.codespellrc

```diff
@@ -2,5 +2,7 @@
 skip = *.js,*.svg,*.eps,.git
 ignore-words-list = fo,te,als
 builtin = clear,rare
-dictionary = .codespell_dict
+# this overloads default dictionaries and I have not yet figured out
+# how to have multiple https://github.com/codespell-project/codespell/issues/2727
+# dictionary = .codespell_dict
```

Comment:
Give this a try:
```
dictionary = -
dictionary = .codespell_dict
```
Not really intuitive: codespell should probably accept comma-separated dictionaries.